tests/requirements.txt

```diff
@@ -1,7 +1,11 @@
-pytest>=6.1.1,<7.0.0
-ledgerwallet>=0.1.2
+# pytest>=6.1.1,<7.0.0
```

Comment:
I'll clean this up before merge. Forgot to remove commented obsolete requirements.

Reply:
Done.